apps/openmw/mwmechanics/aisequence.cpp

```diff
@@ -366,6 +372,18 @@ void AiSequence::readState(const ESM::AiSequence::AiSequence &sequence)
     if (!sequence.mPackages.empty())
         clear();
 
+    // If there is more than one non-combat, non-pursue package in the list, enable repeating.
+    for (std::vector<ESM::AiSequence::AiPackageContainer>::const_iterator it = sequence.mPackages.begin();
+         it != sequence.mPackages.end(); ++it)
```

Comment:
This doesn't quite do what the comment says. Imagine a sequence of 1. Follow, 2. Combat. The code would set mRepeat to true, but it shouldn't.